pkg/operator/operator.go

```diff
@@ -1187,17 +1188,11 @@ func newRunReportForError(reason string, err error) runReport {
 	}
 }
 func (r runReport) isDegraded() bool {
-	if isNilOrAsExpected(r.degraded) {
-		return false
-	}
-	return true
```

Comment:
This and the change to `isUnavailable` is already part of #1979. Do we need this here?

Reply:
Probably rebased at some point, I'll check but it won't do any harm :D